integration/README.md

```diff
@@ -2,10 +2,18 @@
 
 These are VM based tests for core u-root functionality such as:
 
--   retrieving and kexec'ing a Linux kernel,
--   DHCP client tests,
--   uinit (user init), and
--   running unit tests requiring root privileges.
+* retrieving and kexec'ing a Linux kernel,
+* DHCP client tests,
+* uinit (user init), and
+* running unit tests requiring root privileges.
+
+All tests are in the integration/ directory. Within that, there are a
+few subdirectories:
```

Comment:
What is the reasoning behind splitting the tests into two subdirectories?

Reply:
I initially thought I'd modify the tests so that everything in the generic-tests directory could just use the generic initramfs. Later since I was keeping the option for a custom uinit I didn't end up doing that but kept the different directories in case going forward, we wanted to separate tests eg. boot vs. generic vs. go tests. Its not necessary though